source/addComment.js

```diff
@@ -1,3 +1,22 @@
 $(document).ready(function(){
-  // add your code here
+  $('#comments #new_comment_button').on('click', function(e) {
+    e.preventDefault();
+    var newCommentButton = $('#comments #new_comment_button');
+    newCommentButton.detach(); 
+    $('#comments #comment_list').append('<li id="new_comment">\n</li>');
+    $('#comment_list #new_comment').append('<form></form>');
+    $('#new_comment form').append('<input type="text" placeholder="Comment" id="new_comment_body">');
+    $('#new_comment form').append('<input type="text" placeholder="Author" id="new_comment_author">');
+    $('#new_comment form').append('<input type="submit" value="Submit" id="new_comment_submit">');
+    document.getElementById('new_comment_submit').addEventListener('click', function(e) {
+      e.preventDefault();
```

Comment:
You shouldn't have to `preventDefault` here. It is useful on submit as you don't want the page to reload but it doesn't help us with `click` events